### PR TITLE
Viewset guidelines

### PR DESCRIPTION
**Pull request by piyapan039285**
* add readme instruction guidelines
* add GenericView topic guidelines
* correct syntaxes and grammars
* set linting `MD034` to false because we need to provide api url in bare url in sample readme.

### CODE REVIEW

File: django/readme/sample.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+```sh
+make run-django
+```
+
+# Accounts
+
+## API
+
+* url: http://localhost:8000/
```
Comment (powerdefy) [inline django/readme/sample.md]:
```suggestion
* url: `http://localhost:8000/`
```

You don't need to disable the rule krub.

File: .markdownlint.yml [hunk 2]
```diff
@@ -87,7 +87,7 @@ MD032: true
 MD033: false
 
 # Bare URL used <https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md034>
-MD034: true
+MD034: false
```
Comment (powerdefy) [inline .markdownlint.yml]:
```suggestion
MD034: true
```
Per my below comment you don't need to disable this rule.

**Review by powerdefy — CHANGES_REQUESTED** [review]
I think you dont need to disable the rule krub.

**Review by earthpyy — APPROVED** [review]
Cool krubb

**Review by powerdefy — APPROVED** [review]
Good krub

**Pull request merged**